.github/workflows/release-subsquid-processor.yml

```diff
@@ -0,0 +1,64 @@
+name: "Release Subsquid Production Image"
+env:
+  SUBSQUID_IMAGE_RELESE_TAGS: ${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:latest,${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:{{ github.event.pull_request.head.sha }},v0.0.1
+on:
+  push:
+    tags:
+      - 'subsquid-processor-v[0-9]+\.[0-9]+\.[0-9]+'  # e.g. subsquid-processor-v0.0.1
+
+jobs:
+  test-subsquid:
+    name: Test Subquid
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+      - name: Setup jest
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm ci
+        working-directory: subsquid
+      - name: Run tests
+        run: npm run test
+        working-directory: subsquid
+
+  subsquid-processor:
+    name: "Package Subsquid Processor"
+    needs:
+      - test-subsquid
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    concurrency:
+      group: ${{ github.workflow }}-package-subsquid-processor-${{ github.event.pull_request.title }}
+      cancel-in-progress: true
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
```

Comment:
We have a template for this :)

Reply:
again, I copied from `build-test-deploy.yml` :) so this means that it should be fixed there also

Reply:
I can't use https://github.com/ComposableFi/composable/blob/main/.github/templates/docker-publish/action.yml
because    [ # TODO: nixify these test steps, and add me to subsquid-processor](https://github.com/ComposableFi/composable/blob/main/.github/workflows/build-test-deploy.yml#L1002)

Reply:
Alrighty sir; my memory is probably foggy.